File: js/sumPairs.js
exports.sumPairs = function(arr, num) {
    ansArr = []
    for (let i = 0; i < arr.length; i++) {
    arr[i]
        for (let j = 0; j < arr.length; j++) {
            arr[j]
            if(arr[i] + arr[j] === num) {
                ansArr.push([arr[i], arr[j]])
                arr[i] = -1
                arr[j] = -1
            }
        }
    }

    if (ansArr.length === 0) {
        return 'unable to find pairs'
    } 
    return ansArr      
};

File: python/sum_pairs.py
def sum_pairs(num_list, target):
    #ans list to log match pairs
    ans_list = []
    #nested for loop looping through summing each value to each other value in the list
#   #and then comparing to target, if match then pushing into ans list
    for i in range(0, len(num_list), 1):
        for j in range(1, len(num_list), 1):
            if num_list[i] + num_list[j] == target:
                ans_list.append([num_list[i], num_list[j]])
                num_list[i] = -1
                num_list[j] = -1
    #if nothing in ans list returning unable, else returning the ans list
    if len(ans_list) == 0:
        return 'unable to find pairs'
    else:
        return ans_list                
